Libplanet.Stun/Stun/Messages/StunMessage.cs

```diff
@@ -24,7 +24,9 @@ protected StunMessage()
             TransactionId = transactionId;
         }
 
+
         // TODO Should document following STUN / TURN RFC
```

Comment:
Is it still needed?

Reply:
It was my mistake 😥 I will remove that. The CI build seems failed by that.

Reply:
It still is here 💦

Reply:
Oh oops 😨 
In this case, it seems there is some problem on ice-server.

Reply:
> Is it still needed?

I left the line because there are some methods undocumented yet, like `StunMessage.Encode()`. Is it better to document those in this PR?

Reply:
I see. I guess it's okay that we document them in the next PR.